source/code/python/authentication/mfa_challenge_sms_factor_req.py

```diff
@@ -0,0 +1,5 @@
+# Setting a message is optional, set a custom message only if you want to
+# overwrite the default message. Make sure you use a "${code}" placeholder
+# in your custom message.
+challenge = sms_factor.challenge_factor(
+    message='For the sake of example, your code is ${code}')
```

Comment:
Let's get this on one line as well. It'll render nicer that way in the Sphinx markup, and be easier to copy+paste.